spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+package SPDM is
+
+   -- Security Protocol and Data Model (SPDM) Specification
+   -- Document Identifier: DSP0274
+   -- Version: 1.1.0
+
+   -- We will define a static hash type Hash, even though this is acutally based on prior
+   -- negotiation. We will use Hash'Size throughout this spec to simulate that.
+   -- ISSUE: Componolit/systematization-binary-schemas#53
+   type Hash is
+      message
+         Data : Opaque
+            with Size => 8 * 48;
+      end message;
```

Comment:
That is true for this concrete case. I think it would make sense to at least replace all references to systematization repositories which will not be replaced in #92.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.